packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -394,6 +401,39 @@ ListButton.propTypes = {
   }).isRequired,
 };
 
+const LinkButton = () => {
+  const editor = useSlate();
+
+  const isLinkActive = isBlockActive(editor, (node) => node.type === 'link');
```

Comment:
Hey @Feranchz , in my Pr https://github.com/strapi/strapi/pull/18181 I removed the definition of isBlockActive because I found a different way to solve my issue with mixed blocks selection in the editor, my suggestion is to move the definition of isBlockActive in the component where you need it, find a better name for it and also you can find the matchType definition directly here https://github.com/strapi/strapi/blob/e7e82d57df7e80015b9e78fb0ff8ea2d0656d049/packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js#L366 so you can just use the hook to retrieve the matchNode function for the link, thanks

Reply:
hey @Feranchz my PR is merged now